libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.ts

```diff
@@ -74,23 +80,26 @@ import { DhUsersTabSearchComponent } from './dh-users-tab-search.component';
     TranslocoModule,
     WattSpinnerModule,
     WattCardModule,
-    DhUsersTabSearchComponent,
     DhUsersTabTableComponent,
+    DhUsersTabSearchComponent,
+    DhUsersTabStatusFilterComponent,
     DhSharedUiPaginatorComponent,
     DhUsersTabGeneralErrorComponent,
   ],
 })
 export class DhUsersTabComponent {
-  private readonly store = inject(DhAdminUserManagementDataAccessApiStore);
+  readonly users$ = this.store.users$;
+  readonly totalUserCount$ = this.store.totalUserCount$;
 
-  users$ = this.store.users$;
-  totalUserCount$ = this.store.totalUserCount$;
+  readonly pageIndex$ = this.store.paginatorPageIndex$;
```

Comment:
Question: Why readonly? Observables are by nature one directional? Is it because we dont want them to be reassigned?

Reply:
Yes, that was the idea.